### ISSUE TITLE
Missing training procedure

### ISSUE THREAD

**Issue by htalibart** [atzkenneth/dragonfly_gen#8]
Hello,

Impressive paper!

I was wondering if you were planning on making all of your code public, including the training procedure? How were your models trained precisely, what were the inputs and outputs, how was the loss computed exactly? The supplementary information only says that the loss was "cross-entropy" but as indicated in your paper there are multiple desired outputs (for example because there are multiple known ligands for one input pocket), so how was it handled?

Thank you in advance!
